nym-vpn-android/nym-vpn-client/src/main/java/net/nymtech/vpn/util/Constants.kt

```diff
@@ -134,4 +134,34 @@ object Constants {
 		Os.setenv("NYXD", "https://canary-validator.performance.nymte.ch", true)
 		Os.setenv("NYM_API", "https://canary-api.performance.nymte.ch/api", true)
 	}
+
+	fun setupEnvironmentQA() {
+		setupCommon()
+
+		Os.setenv("REWARDING_VALIDATOR_ADDRESS", "n1rfvpsynktze6wvn6ldskj8xgwfzzk5v6pnff39", true)
```

Comment:
FYI: what I'm working is that soon you'll be able to query all these at runtime instead of hardcoding them